File: 프로그래머스/1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.java
import java.util.*;
class Solution {
    public int[] solution(int[] lottos, int[] win_nums) {
        int[] ans = {6,6,5,4,3,2,1};
        int cnt_0 = (int) Arrays.stream(lottos).filter(x -> x == 0).count();
        int c = 0;
        for (int l : lottos) {
            for(int n : win_nums) {
                if (n == l) c++;
            }
        }
        return new int[] {ans[cnt_0+c],ans[c]};
    }
}

File: 프로그래머스/1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.kt
class Solution {
    fun solution(lottos: IntArray, win_nums: IntArray): IntArray {
        val n = intArrayOf(6,6,5,4,3,2,1)
        val cnt_0 = lottos.count { it == 0 }
        var c = 0
        for (l in lottos) {
            if (l in win_nums) c++
        }
        return intArrayOf(n[cnt_0+c],n[c])
    }
}